fbchat/models.py

```diff
@@ -456,12 +458,15 @@ class LocationAttachment(Attachment):
     image_height = None
     #: URL to Bing maps with the location
     url = None
+    # Address of the location
+    address = None
 
-    def __init__(self, latitude=None, longitude=None, **kwargs):
+    def __init__(self, latitude=None, longitude=None, address=None, **kwargs):
```

Comment:
I think `latitude` and `longitude` are available in the constructor because they have a relevance if you send the location using `Client.sendLocation` or likewise, but there's no reason for ever setting `address` yourself.

But this is super nitpicky, and the described pattern isn't consistent throughout the rest of the code, so it probably doesn't really matter 😁

Reply:
I figured if some message locations only have addresses and no co-ords that it must mean they're also able to be constructed in this way (by sending a recogniseable address, not just co-ords) so I wanted to account for that. If it's not the case, then what you're saying makes sense. I just don't know why Facebook would then return a location without co-ords, but only the address.

Reply:
Oh yeah, that does make sense. Perhaps the task is then to update `Client.sendLocation` instead. But that'll probably be really difficult to figure out

Reply:
Maybe not 'instead', maybe 'as well as'. This change doesn't break client.sendLocation - but it does fix the receipt of messages with locations that don't have co-ords. 
client.sendLocation still functions, just limited to sending co-ords. Adding functionality to send an address (that Facebook accepts) seems a separate issue to me... :)

Reply:
Yeah, sorry, I meant "as well as" 😉. And yeah, it's a separate issue!